core/domain/exp_services.py

```diff
@@ -1220,11 +1576,28 @@ def get_search_rank_from_exp_summary(exp_summary):
 
 
 def get_search_rank(exp_id):
+    """Returns the search rank.
+
+    Args:
+        exp_id: str. The id of the exploration.
+
+    Returns:
+        int. The rank of the exploration.
+    """
     exp_summary = get_exploration_summary_by_id(exp_id)
     return get_search_rank_from_exp_summary(exp_summary)
 
 
 def _exp_to_search_dict(exp):
+    """Updates the dict to be returned with, if the given exploration
+    is to be indexed for further queries or not.
+    Args:
+        exp: Exploration. Exploration domain object.
+
+    Returns:
+        dict. The representation of exploration to be used in the search
```

Comment:
The representation of the given exploration, in a form that can be used by the search index.